components/tools/risk-survey.tsx

```diff
@@ -60,50 +61,45 @@ const RiskSurvey = ({ submissionCount }: { submissionCount: number }) => {
     <Formik initialValues={initialValues} onSubmit={handleSubmit}>
       {(formik) => (
         <form className="w-full h-full" onSubmit={formik.handleSubmit}>
-          <Flex
-            flexDir={"column"}
-            gap={4}
-            w={"full"}
-            bg={"lily-white.100"}
-            shadow={"5px 5px 0 black"}
-            borderColor={"black"}
-            borderWidth={2}
-            p={6}
-          >
-            <Box>
-              <Text fontWeight={500} fontSize={"2xl"} textAlign={"center"}>
+          <CardContainer>
```

Comment:
@ngd-adams you happy with `CardContainer` here? Or have you not pushed the latest code yet?

Reply:
I will push now. Made card container and card variants different